src/pure_pursuit.cpp

```diff
@@ -103,36 +105,38 @@ double PurePursuit::path_length()
 {
 	double totalLength = 0;
 
-	for (uint32_t i = 0; i < m_robot_path.size() - 1; i++)
-	{
+	for ( int i = 0; i < ( m_robot_path.size() - 1 ); i++ )  // will use current point (i) and next point (i+1)
+	{            																						// until i = the second to last point in vector
```

Comment:
this comment got super messed up, could you move the whole comment before the for loop so its not weird?

Reply:
yeah. I haven't made it to cleaning up the library yet but when i do, i'll do that